### PR TITLE
FIX: Adjust docker apache config to serve from public directory

### PR DESCRIPTION
**Pull request by crosa7**
## Description
Adjustment to make apache serve from the public directory instead of root folder

### CODE REVIEW

**Review by mychidarko — COMMENTED** [review]
This will also try to serve from the public directory even in non-mvc projects. Can this be made into a different docker theme, so we have one for base projects and the other for Leaf MVC and Leaf API

**Comment by crosa7** [comment]
> This will also try to serve from the public directory even in non-mvc projects. Can this be made into a different docker theme, so we have one for base projects and the other for Leaf MVC and Leaf API

Good point! Did not think of this. Will do that, thanks for the feedback!

**Comment by crosa7** [comment]
> This will also try to serve from the public directory even in non-mvc projects. Can this be made into a different docker theme, so we have one for base projects and the other for Leaf MVC and Leaf API

Changed thanks!

File: src/CreateCommand.php [hunk 1]
```diff
@@ -181,7 +181,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         if ($process->isSuccessful()) {
             if ($this->getAppDockPreset($input, $output)) {
-                FS::superCopy(__DIR__ . '/themes/docker', $directory);
+                $dockerThemeFolder = __DIR__ . '/themes/docker';
+
+                if ($preset === 'mvc') {
```
Comment (mychidarko) [inline src/CreateCommand.php]: Or if the preset is 'api' since that would also need the MVC preset

Comment (mychidarko) [reply src/CreateCommand.php]: Here @crosa7

Comment (crosa7) [reply src/CreateCommand.php]: Added, thank you!

**Comment by mychidarko** [comment]
Awesome, thanks

**Review by mychidarko — APPROVED** [review]

**Pull request merged**